src/analyzer.ts

```diff
@@ -111,40 +127,38 @@ export class StreamAnalyzer extends Transform {
     return this._dependenciesProcessingStream;
   }
 
-  _transform(file: File, encoding: string, callback: FileCB): void {
+  async _transform(file: File, encoding: string, callback: FileCB): Promise<void> {
```

Comment:
why is this `async`? `_transform` isn't intended to return anything. I'd consider using an AIIFE below to demark the async part of the work and use `await`.